feature-libs/quote/components/list/quote-list.component.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { QuoteState } from '@spartacus/quote/root';
+import { ICON_TYPE } from '@spartacus/storefront';
+import { QuoteListComponentService } from './quote-list-component.service';
+
+@Component({
+  selector: 'cx-quote-list',
+  templateUrl: './quote-list.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class QuoteListComponent {
+  protected quoteListComponentService = inject(QuoteListComponentService);
+
+  sorts = this.quoteListComponentService.sorts;
+  sortLabels$ = this.quoteListComponentService.sortLabels$;
+  quotesState$ = this.quoteListComponentService.quotesState$;
+  dateFormat: string = 'MMMM d, YYYY h:mm aa';
+  iconTypes = ICON_TYPE;
+
+  constructor() {
+    this.changePage(0);
+    this.changeSortCode('byCode');
+  }
+
+  changeSortCode(sortCode: string): void {
+    this.quoteListComponentService.setSort(sortCode);
+  }
+
+  changePage(page: number): void {
```

Comment:
Please, add a comment

Reply:
@ChristophHi, could you please add a comment to the open API?

Reply:
Comment already present in quote-list-component-service